### ISSUE TITLE
Calculate score according to the time spent by player

### ISSUE THREAD

**Issue by bhansa** [bhansa/fireball#15]
Add a score count for the time spent by the player, and later will add conditions to restrict that. For now, just add the score count, which is just how much time spent.

**Comment by ghost** [comment]
We can use the timer variable introduced in pull request #12 for this, right? Maybe multiplying the value with a suitable factor before displaying it will also be useful.

**Comment by bhansa** [comment]
Yes, we can make use of that. Let me also create an issue for a `start` button during start of the game so it's easy to trigger the time variable value. What do you say @amruchand ?

**Comment by ghost** [comment]
That would be nice... Maybe we should also implement a "reset" button as well to zero the timer and clear the score?

**Comment by bhansa** [comment]
That sounds good, I am adding you as a collaborator for this project :) So you can also start creating issues.
